### PR TITLE
Feat/update colors to fix accessibility issues

### PR DESCRIPTION
**Pull request by nicolaskempf57**
Some accessibility issues from our report are not really about colors but about our UI. Our search inputs doesn't have any background or border so the only way to see them is the placeholder or the focus ring. Do we want to do these changes in this PR ?

Some changes are really important for a user point of view so I added them here :
- code in markdown is shown with a black background due to its orange color. We can use something else if we want to keep a light background
- some orange text are now using text-default-warning or text-default-error instead
- green on our home is now green-400 due to contrast issues
- light grey btn-tab on home are now blue. We may want to iterate on this to improve the difference between selected tab and others

Some changes we might want to consider :

- ~move to DSFR breadcrumbs instead of the current color change~ (left for another PR)
- ~move .btn-tab to DSFR tags with custom active style (see https://www.gouvernement.fr/toute-l-actualite for an example)~(done in https://github.com/etalab/udata-front/pull/57)
- update resource background from @grey-50 to @background-contrast-grey to match other backgrounds (summary and read-more)
- ~merge @grey-50 and @grey-100 together (grey-100 is the same grey as background-contrast-grey)~(left for another PR)

Close https://github.com/etalab/data.gouv.fr/issues/584
Close https://github.com/etalab/data.gouv.fr/issues/625
Close https://github.com/etalab/data.gouv.fr/issues/359
Close https://github.com/etalab/data.gouv.fr/issues/268
Close https://github.com/etalab/data.gouv.fr/issues/376

### CODE REVIEW

File: theme/js/components/dataset/resource/resource.vue [hunk 1]
```diff
@@ -17,44 +17,42 @@
...
-                        <a
-                          href="#"
+                        <button
                           @click.prevent="expand"
                           role="button"
                           :aria-expanded="expanded"
                           :aria-label="$t('See more details')"
                           :aria-controls="'resource-' + resource.id"
-                          class="accordion-button"
-                          v-html="ChevronIcon"
```
Comment (maudetes) [inline theme/js/components/dataset/resource/resource.vue:36]: We don't need to import `ChevronIcon` here anymore.

File: udata_front/theme/gouvfr/templates/svg/chevron.svg [hunk 2]
```diff
@@ -1,3 +1,3 @@
 <svg aria-hidden="true" width="6" height="9" viewBox="0 0 6 9" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M5.25 4.89844C5.46094 4.6875 5.46094 4.33594 5.25 4.10156L2.0625 0.914062C1.82812 0.703125 1.47656 0.703125 1.26562 0.914062L0.726562 1.45312C0.515625 1.6875 0.515625 2.03906 0.726562 2.25L3 4.52344L0.726562 6.77344C0.515625 6.98438 0.515625 7.33594 0.726562 7.57031L1.26562 8.08594C1.47656 8.32031 1.82812 8.32031 2.0625 8.08594L5.25 4.89844Z" fill="#198CF2"/>
```
Comment (maudetes) [inline udata_front/theme/gouvfr/templates/svg/chevron.svg:2]: I guess we use chevron only in some places left, we'll be able to remove it later :)

**Review by maudetes — COMMENTED** [review]
:star2:

Thank you for this important PR ! I didn't find much unwanted side effects, it appears that you've checked everything thoroughly already.
Let's wait for some feedback by the team if we missed some things.

File: theme/js/components/search/suggestor.vue [hunk 3]
```diff
@@ -50,8 +50,8 @@ Simply provide the many necessary props :
     object
     ref="multiselect"
   >
-    <template v-slot:multiplelabel="{ values }">
-      <div class="suggestor-labels" v-if="!opened">
+    <template #multiplelabel="{ values }">
+      <div class="multiselect-multiple-label" v-if="!opened">
```
[the remark this answers is not in the document]
Comment (nicolaskempf57) [reply theme/js/components/search/suggestor.vue:54]: Can you try to force the npm install with `npm ci` before the `inv assets-watch` ? I updated vue-multiselect with the last commit

File: theme/js/components/search/suggestor.vue [hunk 4]
```diff
@@ -96,82 +95,92 @@ export default {
...
+      options: null, // Current options list, same structure as `value` above (but contains all possible options). Can be dynamic (async)
     };
   },
   watch: {
-    values: function (value) {
-      //This lib doesn't support `undefined` values and will not reset its state.
-      //This allows to reset the value if the parent component decides to clear the value.
+    opened(value) {
+      if(value) {
+        this.$refs.multiselect.$refs.input.placeholder = this.searchPlaceholder || this.placeholder;
```
[the remark this answers is not in the document]
Comment (nicolaskempf57) [reply theme/js/components/search/suggestor.vue:120]: With our previous version of vue-multiselect, the <input> was hidden when the select was closed and both the Multiselect placeholder and the input placeholder was shown when the select was opened.

Now, this is the opposite, so  the <input> placeholder attribute is shown at the same place as the <Multiselect> placeholder attribute when the select is closed.

File: theme/js/components/search/suggestor.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (maudetes) [reply theme/js/components/search/suggestor.vue:54]: Ah yes, my mistake! :pray:

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,8 @@
 
 - Change urls in Participate banner to relevant etalab guides [#53](https://github.com/etalab/udata-front/pull/53)
 - Update DSFR to v1.2.1 [#45](https://github.com/etalab/udata-front/pull/45)
+- :warning: @bg-beige is remove, use @background-contrast-grey instead
+- Update colors to fix accessibility issues [#45](https://github.com/etalab/udata-front/pull/45)
```
[the remark this answers is not in the document]
Comment (nicolaskempf57) [reply CHANGELOG.md]: Done in commit 5bd0fa8

File: udata_front/theme/gouvfr/templates/svg/chevron.svg (hunk 2 above)
Comment (nicolaskempf57) [reply udata_front/theme/gouvfr/templates/svg/chevron.svg:2]: I added the DSFR icon in post/display, it was the last one to use this svg.

File: theme/js/components/search/suggestor.vue (hunk 4 above)
[the remark this answers is not in the document]
Comment (maudetes) [reply theme/js/components/search/suggestor.vue:120]: Thank you for the explanation! I hadn't correctly use the updated vue-multiselect.
However, I have this rendering on select (the text input is smaller than the filter box).
image

Additionally, I don't see the `clear` icon on licenses when one is selected (whereas it appears correctly on organization filter).

[the remark this answers is not in the document]
Comment (nicolaskempf57) [reply theme/js/components/search/suggestor.vue:121]: Done !

[the remark this answers is not in the document]
Comment (nicolaskempf57) [reply theme/js/components/search/suggestor.vue:121]: The clear icon is still not working for licenses. I'll try to investigate and see if it's related to https://sentry.data.gouv.fr/organizations/sentry/issues/12597/?project=5

[the remark this answers is not in the document]
Comment (nicolaskempf57) [reply theme/js/components/search/suggestor.vue:125]: Fixed.

**Review by maudetes — APPROVED** [review]
I think we're all good ! :) Thank you !

Do you want to cross-validate new colors (especially the cumulus blue) with the team or not?

**Pull request merged**